pages/04.Reference/06.images/docs.en.md

```diff
@@ -65,6 +69,16 @@ data "openstack_images_image_v2" "ubuntu-bionic" {
 }
 ```
 
+### Known issues with community images
+
+- Debian Buster community OpenStack image has a bug in `ifupdown` configuration. After initial boot, eth0 interface
+will be configured with both DHCP and static cloud-init configuration. Since both will try to assign the same IP
+address to eth0, `RTNETLINK answers: File exists` error will be produced, resulting `networking.service` systemd unit
+in errored state. This will block DHCP client from refreshing lease after 24 hours. Solution is to remove dynamic eth0
+configuration from `/etc/network/interfaces`, or to remove static configuration written by cloud-init from
+`/etc/network/interfaces.d`. After this change, restart of `systemd-networkd` systemd unit, or restart of virtual
+machine is required.
```

Comment:
do you think we can link some upstream bug ticket or something?

Reply:
It is just an ignored pull request in salsa.debian.org, and actual bug report was sent to mailing list and never actually appeared in it, so mailing list is broken too. Not sure if we shall provide some links. In case it is fixed, I'll rephrase the text, so it is saying that debian buster MAY has this issue.